### PR TITLE
Update MediaCompose to build with latest release of Unbundled AAOS

### LINKED ISSUE #67: MediaCompose is incompatible with the current release of Unbundled AAOS
Error:
```
e: file:///usr/local/google/home/froeht/git/designcompose/reference-apps/aaos-unbundled/standalone-projects/mediacompose/media/src/main/java/com/android/designcompose/reference/media/MediaAdapter.kt:467:39 Unresolved reference: artist
e: file:///usr/local/google/home/froeht/git/designcompose/reference-apps/aaos-unbundled/standalone-projects/mediacompose/media/src/main/java/com/android/designcompose/reference/media/MediaAdapter.kt:468:38 Unresolved reference: albumTitle
```

We can continue using the previous release. To fetch a clean copy of it run this:
```
    HOST=https://android.googlesource.com/platform/superproject
    BRANCH=ub-automotive-master-20230303
    git clone "$HOST" "$BRANCH" --branch="$BRANCH" \
    --single-branch --filter=blob:none --also-filter-submodules \
    --recurse-submodules --jobs=4
```

### PR DESCRIPTION
**Pull request by rylin8**
Also updated agp plugin version to be compatible with unbundled aaos.

Fixes #67

### CODE REVIEW

**Comment by rylin8** [comment]
I updated to the latest unbundled aaos repo and tested this change. I needed to update unbundledAAOSAndroidGradlePluginVer to make it build. When testing with Spotify, I noticed that the now playing data never showed the artist or album anymore. I checked with Arnaud and he had me verify with some other sources that it does in fact work, and commented that Spotify is likely not populating the artist and album fields properly.

**Review by timothyfroehlich — CHANGES_REQUESTED** [review]
Changes look good to me, but we need to update the tests and doc to stop pulling down the old release. Change all the instances of "ub-automotive-master-20230303" in the repo to just "master".

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -178,7 +178,7 @@ jobs:
     - name: Check out the Unbundled AAOS Repo
       run: |
         HOST=https://android.googlesource.com/platform/superproject
-        BRANCH=ub-automotive-master-20230303
+        BRANCH=ub-automotive-master
```
Comment (iamralpht) [inline .github/workflows/main.yml:181]: Do we want to just use master, or do we want to pin to a particular release again? (This looks like the most recent: https://android.googlesource.com/platform/manifest/+/refs/tags/ub-automotive-master-20230622 )

Comment (rylin8) [reply .github/workflows/main.yml:181]: Given that master is failing to build here (though it builds on my machine with a sync from yesterday), I'll give this a try

Comment (iamralpht) [reply .github/workflows/main.yml]: Sounds good -- looks like the CI checks passed using the tag, too.

**Review by timothyfroehlich — APPROVED** [review]

**Pull request merged**